src/core/src/op/shuffle_channels.cpp

```diff
@@ -2,49 +2,43 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "ngraph/op/shuffle_channels.hpp"
+#include "openvino/op/shuffle_channels.hpp"
 
 #include <numeric>
-#include <shuffle_channels_shape_inference.hpp>
 
 #include "itt.hpp"
-#include "ngraph/attribute_visitor.hpp"
-#include "ngraph/builder/reshape.hpp"
-#include "ngraph/runtime/host_tensor.hpp"
-#include "ngraph/type/element_type.hpp"
-#include "ngraph/type/element_type_traits.hpp"
-#include "openvino/core/validation_util.hpp"
+#include "openvino/core/attribute_visitor.hpp"
 #include "openvino/reference/shuffle_channels.hpp"
+#include "shuffle_channels_shape_inference.hpp"
+#include "validation_util.hpp"
 
-using namespace std;
-using namespace ngraph;
-
-op::ShuffleChannels::ShuffleChannels(const Output<Node>& data, const int64_t axis, const int64_t group)
+namespace ov {
+namespace op {
+namespace v0 {
+ShuffleChannels::ShuffleChannels(const Output<Node>& data, const int64_t axis, const int64_t group)
     : Op({data}),
       m_axis(axis),
       m_group{group} {
     constructor_validate_and_infer_types();
 }
 
-bool ngraph::op::v0::ShuffleChannels::visit_attributes(AttributeVisitor& visitor) {
+bool ShuffleChannels::visit_attributes(AttributeVisitor& visitor) {
     OV_OP_SCOPE(v0_ShuffleChannels_visit_attributes);
     visitor.on_attribute("axis", m_axis);
     visitor.on_attribute("group", m_group);
     return true;
 }
 
-size_t op::ShuffleChannels::get_zero_based_axis() const {
+size_t ShuffleChannels::get_zero_based_axis() const {
     const auto input_rank = get_input_partial_shape(0).rank();
     if (input_rank.is_static()) {
-        OPENVINO_SUPPRESS_DEPRECATED_START
-        return ov::normalize_axis(this, m_axis, input_rank);
-        OPENVINO_SUPPRESS_DEPRECATED_END
+        return ov::util::normalize(m_axis, input_rank.get_length());
```

Comment:
This will apply normalization without axis value validation.

Reply:
Reverted